GLTFSDK/GLTFSDK.CPP.autopkg

```diff
@@ -48,111 +48,87 @@ nuget {
             "${BASE}\readme.md"
         };
 
-        symbols += {
+        side-by-side-symbols += {
```

Comment:
Better to just keep calling it "symbols" as that is what this is.  It is the definition of what to do with things in the "symbols" groups.  Then this change is really just about this definition at the top.

Reply:
Alternatively you could remove the "symbols" group entirely and just add the pdb files into the "lib" section as many NuGet packages do.

Reply:
Great minds, I've already tried both of your suggestions and neither worked:
* If I call the group 'symbols' nothing changes - CoApp continues to put pdbs in a separate package. I think that's because it isn't possible to modify default groups - it's only possible to define new ones (so the old `symbols += {...` bit didn't actually do anything).
* Using the 'lib' group also modifies the targets files such that the pdb file is included in the `<AdditionalDependencies>` list.

Reply:
Le sigh.